.github/workflows/build_cmake.yaml

```
@@ -314,3 +314,25 @@ jobs:
+
+  sonar-cloud:
+    name: SonarCloud Analyzer
+    # This job should be run only for the original repo (either pushes or pull requests originating from the same repo), forks are not allowed.
+    # Workflows initiated by Dependabot do not have access to repo secrets, so this job should not be run in this case.
+    if: >-
+      ${{
+          github.repository == 'ToymanInteractive/toygine2'
+          && ( github.event_name == 'push' || ( github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository ) )
+          && github.actor != 'dependabot[bot]'
+       }}
+    runs-on: ubuntu-26.04
```

actionlint (1.7.12)

[error] label "ubuntu-26.04" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)